### PR TITLE
Update docs to OpenProject use cases

### PR DESCRIPTION
**Pull request by HDinger**
- [x] Replace links from primer repo to ours
- [x] Remove GitHub-only parts
- [x] Explain our angular octicon component 
- [x] Remove jekyll and react from github actions

### CODE REVIEW

File: lib/octicons_helper/README.md [hunk 1]
```diff
@@ -10,12 +10,12 @@ This rails helper lets you easily include svg octicons in your rails apps.
...
     ```
 
 3. Use this tag in your erbs
 
     ```rb
     <%= octicon "alert", :height => 32,  :class => "right left", :"aria-label" => "hi" %>
     ```
-We recommend including the CSS in the [`@primer/octicons`](/packages/javascript) npm module. You can also npm install that package and include `build/build.css` in your styles.
\ No newline at end of file
+We recommend including the CSS in the [`@openproject/octicons-angular`](/packages/javascript) npm module. You can also npm install that package and include `build/build.css` in your styles.
```
Comment (HDinger) [inline lib/octicons_helper/README.md:21]: The mentioned file `build/build.css` does not exist in our package. Is that intended?

File: lib/octicons_node/README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
 ```shell
-npm install @primer/octicons
+npm install @openproject/octicons-angular
 ```
 
 ## Usage
 
-For all the usages, we recommend using the CSS located in [`build/build.css`](https://unpkg.com/@primer/octicons/build/build.css). This is some simple CSS to normalize the icons and inherit colors.
+For all the usages, we recommend using the CSS located in [`build/build.css`](https://unpkg.com/@openproject/octicons-angular/build/build.css). This is some simple CSS to normalize the icons and inherit colors.
```
Comment (HDinger) [inline lib/octicons_node/README.md:16]: Same here, the `build/build.css` does not exists (and the URL is showing nothing). Should we add that?

**Comment by HDinger** [comment]
Hi @b12f 

could you please check the current state of the documenation and add if something is missing? I did not write any parts about the new component in the angular package and how to use it. 
Further, we can discuss whether we want to keep the jeckyll and react parts, although we don't use them.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,30 +1,6 @@
...
-- Once design has been started on an icon, the request issue will be moved to the **Design in Progress** column of the Primer Teams Backlog.
-- Designers should design the icon in Figma and when ready for review, use the [Octicons Push plugin](https://www.figma.com/community/plugin/825432045044458754/Octicons-Push) to create a PR
-  - After a PR is created link to the PR in the icon request issue. PRs need approval from the icon requestor (stakeholder) and at least one designer on the octicons maintainer team.
-
-### 4. Icon request completed
-- When an icon request PR has been approved, communicate that in the issue.
-- After the Octicons release, the new icons that were added will have their request issues moved to the **Done** column
+# Octicons Contribution Guidelines
 
+Thank you for your interest in contributing to Octicons! If you'd like to submit feedback, a bug, or an idea for improvement, please open a new issue in this repo using the appropriate [issue template](https://github.com/primer/octicons/issues/new/choose).
```
Comment (b12f) [inline CONTRIBUTING.md:3]: Do we want to link to our own repo here?

File: README.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 ![octicons cover dark](https://user-images.githubusercontent.com/54012/138925203-80e1afa1-ba54-4731-9525-3c41186663f9.png#gh-dark-mode-only)
 <br>
 <br>
-<h1 align="center">Primer Octicons</h1>
+<h1 align="center">OpenProject's Primer Octicons</h1>
 
-<p align="center">Octicons are a set of SVG icons built by GitHub for GitHub.</p>
+<p align="center">Octicons are a set of SVG icons built by GitHub extended by OpenProject for OpenProject</p>
```
Comment (b12f) [inline README.md:7]: I'd call this `forked`

File: RELEASE.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Releasing changes
+
+Releases are managed by 🦋 [_Changesets_](https://github.com/atlassian/changesets#documentation) which is a great tool for managing major/minor/patch bumps and changelogs.
+
+We have the [changeset-bot comment on new pull requests](https://github.com/changesets/bot#readme) asking contributors or maintainers to add a changeset file, which will become the markdown supported changelog entry for the change.
```
Comment (b12f) [inline RELEASE.md:5]: I don't think we have this bot right now

File: lib/octicons_node/README.md (hunk 2 above)
Comment (b12f) [reply lib/octicons_node/README.md:16]: This comes from the `octicons_node` package. All it includes is the contents of `lib/octicons_node/index.scss`:

```
.octicon {
  display: inline-block;
  vertical-align: text-top;
  fill: currentColor;
  overflow: visible;
}
```

Not sure how to handle this. We could publish the node package after all, or just do the same sass build step in the angular package and publish the css file there as well. wdyt?

**Review by b12f — APPROVED** [review]

**Pull request merged**